sormas-backend/src/main/java/de/symeda/sormas/backend/common/DeletableAdo.java

```diff
@@ -0,0 +1,31 @@
+package de.symeda.sormas.backend.common;
+
+import javax.persistence.Column;
+import javax.persistence.MappedSuperclass;
+
+import de.symeda.auditlog.api.Audited;
+
+/**
+ * An extension of the {@link AbstractDomainObject} that defines data that is essential to the system.
+ * This data may not be deleted from the system, e.g. because it is
```

Comment:
The comment does not make sense anymore

Reply:
in my opinion comments never make sense, but please feel free to suggest a better coment